src/mmds/src/data_store.rs

```diff
@@ -19,18 +19,13 @@ pub struct Mmds {
 }
 
 /// MMDS version.
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, Deserialize, Serialize)]
 pub enum MmdsVersion {
+    #[default]
```

Comment:
out of scope, but shouldn't the default be V2?

Reply:
Yeah, this surprised me too, but the old manual `Default` implementation returned `V1`. The `default` impl seemingly is only used in test anyway though